compiler/optimizer/RedundantAsyncCheckRemoval.cpp

```diff
@@ -420,13 +420,21 @@ bool TR_RedundantAsyncCheckRemoval::callDoesAnImplicitAsyncCheck(TR::Node *callN
        (symbol->getRecognizedMethod()==TR::java_lang_Integer_rotateLeft) ||
        (symbol->getRecognizedMethod()==TR::java_lang_Long_rotateLeft) ||
        (symbol->getRecognizedMethod()==TR::java_lang_Integer_rotateRight) ||
-       (symbol->getRecognizedMethod()==TR::java_lang_Long_rotateRight) ||
-       (symbol->getRecognizedMethod()==TR::sun_misc_Unsafe_compareAndSwapInt_jlObjectJII_Z) ||
-       (symbol->getRecognizedMethod()==TR::sun_misc_Unsafe_compareAndSwapLong_jlObjectJJJ_Z) ||
-       (symbol->getRecognizedMethod()==TR::sun_misc_Unsafe_compareAndSwapObject_jlObjectJjlObjectjlObject_Z)
+       (symbol->getRecognizedMethod()==TR::java_lang_Long_rotateRight)
        )
        return false;
 
+   // In Java9 we use the same enum values for both the sun.misc.Unsafe wrappers
```

Comment:
The "Java9" comments seem very point-in-time to me.  Will someone reading these comments 5 years from now think this only applies to Java9?   Perhaps saying "Beginning in Java9..." might be better.